tests/vmi_multiqueue_test.go

```diff
@@ -123,5 +123,35 @@ var _ = Describe("[Serial]MultiQueue", func() {
 			}
 		})
 
+		It("should be able to create a multi-queue VMI when requesting a single vCPU", func() {
+			vmi := tests.NewRandomFedoraVMIWitGuestAgent()
+			multiQueue := true
+
+			// by default, we request a single vCPU
+			vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue = &multiQueue
+			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
```

Comment:
nit: I think this is no longer needed.

Reply:
Would you elaborate more ? What changed so it no longer needed ?

Reply:
The Fedora factory is adding as it is needed on the distribution.

Reply:
I'm not using the fedora factory, I'm using the VM spec that is being used in multi-queue.

Reply:
You are using `NewRandomFedoraVMIWitGuestAgent` which uses it.